lib/license_scout/overrides.rb

```diff
@@ -671,6 +671,7 @@ def default_overrides
         ["debug", "MIT", ["https://raw.githubusercontent.com/visionmedia/debug/master/LICENSE"]],
         ["performance-now", "MIT", ["https://raw.githubusercontent.com/braveg1rl/performance-now/master/license.txt"]],
         ["cli-table", "MIT", ["README.md"]],
+        ["process", "MIT", ["https://raw.githubusercontent.com/defunctzombie/node-process/v0.5.2/LICENSE"]],
```

Comment:
Ugh, sorry about leading you in the wrong direction here. The change you had before was actually better, and I think we should pin to master and not this version. I was under the assumption that we'd be able to pin the actual dependency to a version as well, but it looks as if that's not the case, so let's go more general here and since 0.5.2 and master both have the same license it's going to be safe to point at that one.